src/main/java/sg/ncl/MainController.java

```diff
@@ -953,6 +957,11 @@ public String editAccountDetails(
             errorsFound = true;
         }
 
+        if (errorsFound == false && !editUser.isPasswordValid()) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1125)